.github/workflows/CI.yml

```
@@ -61,7 +61,7 @@ jobs:
       - name: "Run tests"
         uses: julia-actions/julia-runtest@latest
         with:
-          coverage: ${{ matrix.julia-version == '1.6' }}
+          coverage: ${{ matrix.julia-version == '1.8' }}
           depwarn: error
       - name: "Process code coverage"
```

```suggestion
      - name: "Process code coverage"
        if: matrix.julia-version == '1.8'
```
Can you add this to the upload step further down as well?
(This might avoid a bunch of the edits for the codecov comments that often initially claim coverage -75%)